catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java

```diff
@@ -49,6 +50,12 @@ public JdbcTable load(String databaseName, String tableName) throws NoSuchTableE
             String.format("Table %s does not exist in %s.", tableName, databaseName));
       }
       String comment = table.getString("REMARKS");
+      if (StringUtils.isEmpty(comment)) {
+        // In Mysql version 5.7, the comment field value cannot be obtained in the driver API.
+        LOG.warn(
+            "Not found comment in mysql driver api. Will try to get comment from the system table");
```

Comment:
Actually you didn't get "comment" from system table, so you'd better update the log here and method name below.